.gitlab-ci-check-python3-format.yml

```diff
@@ -67,7 +67,7 @@ test:check-python3-formatting:
     - git fetch origin 'refs/heads/*:refs/heads/*'
     # Get last remaining tags, if any.
     - git fetch --tags origin
-    - git clone --depth 1 http://github.com/mendersoftware/mendertesting /tmp/mendertesting
+    - git clone --depth=1 --depth 1 http://github.com/mendersoftware/mendertesting /tmp/mendertesting
```

Comment:
Depth 1. I said **depth 1**!! :smile:

Reply:
@kacf ooops, sorry, I've missed it when checking the result of my find & replace.
Fixed.